docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -92,6 +92,21 @@ Analysis takes minutes to hours to run depending on the size of the application
 ====
 {ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time. With more cluster capacity, more analysis processes can be executed in parallel.
 ====
+
+. Optional: To track the status of your active analysis tasks, open the *Task Manager* tab by clicking the notifications button.
++
+Alternatively, hover over the application name.
++
+The *Task Manager* displays the following information:
+
+** *ID*: The ID of the task.
+** *Application*: The application name associated with the task.
+** *Status*: The status of the task, for example, *Scheduled*, *Pending*, *In progress*, *Succeeded*, or *Failed*.
+** *Kind*: The type of the task, for example, *analyzer* or *discovery*.
+** *Priority*: Priority of the task.
+** *Preemption*: This column shows if the task must be executed prior to any other task. The values are *true* or *false*.
+** *Created By*: The name of the user who created the task.
+
```

Comment:
@mpershina we are on the application inventory page at this step, please remove lines 100-108 which describe the table on the "Task Manager" page

cc: @dymurray @rromannissen to confirm

Reply:
Agreed